### ISSUE TITLE
Macros removed from buttons when using loadWorkbook() on .xlsm file

### ISSUE THREAD

**Issue by notoriousPowell** [ycphs/openxlsx#44]
**Describe the bug**
Hi, 

First off I just want to say that this package is excellent and my bug must be quite a niche use case. Also, I am totally new to this so I apologise in advance if I write anything stupid. 

I am currently working on something where I need to write some data to a .xlsm file, which I am using as a template.  

The .xlsm file has a worksheet containing a simple form with some buttons that are linked to macros. 

After the workbook is loaded into R via loadWorkbook() and then saved using saveWorkbook(), the macro links are getting removed from the buttons in the form - the buttons simply become pictures instead. The macros also don't appear to be listed when you go to Developer -> Macros. I can see that all the VBA code is there, though. 

Looking at the underlying folder structure of the .xlsm file, it looks like the ctrlProps folder is missing from the xl directory:

![xl Directory Comparison](https://user-images.githubusercontent.com/61312073/75030362-dba66280-549b-11ea-8b68-f09ca5937aee.png)

1. Run loadWorkbook() on a .xlsm file with a button linked to a macro
2. Save the workbook to any location using saveWorkbook()
3. Open the .xlsm file output
4. Observe that the buttons have simply become pictures and are no longer linked to macros

**Expected behavior**
Macro links to the buttons are maintained and can be executed

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise

sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17763)
openxlsx version 4.1.4

**Comment by ycphs** [comment]
Hi,

The package currently supports only xlsx files. But it would be great to have this feature. On the short term, I can't implement your request, but I would like to welcome you to contribute to the package.

**Comment by ycphs** [comment]
Can you try to implement it and create a pull request?

**Comment by JanMarvin** [comment]
Could anyone provide any example xlsm file? I tried to create one in LibreOffice (assign a simple macro to a button), but saving that as xlsxm didn't work.

**Comment by scottporter** [comment]
I ran into this same issue, and noticed it was stalled because of lack of an example, so I put one together.

As I was putting together the example, I realized that it is only ActiveX buttons that get broken.  Regular form control buttons or images with linked macros continue to work.  Probably b/c in this case the VBA code located on the Sheet (instead of in a separate module)?

Here are the excel files I created with buttons: 
[spreadsheets_with_buttons.zip](https://github.com/ycphs/openxlsx/files/12545120/spreadsheets_with_buttons.zip)

Only the ActiveX button breaks, the other two will work fine.

Here is the code to replicate the issue with the activeX button:

```
wb <- openxlsx::loadWorkbook("spreadsheet_with_activex_button.xlsm")

data_to_paste <- data.frame(
  A = 3,
  B = 5,
  C = 8,
  D = 13
)

openxlsx::writeData(
  wb = wb,
  sheet = "Sheet1",
  x = data_to_paste,
  startCol = 2,
  startRow = 2,
  colNames = FALSE,
  rowNames = FALSE
)

openxlsx::saveWorkbook(wb,file = "modified_spreadsheet_with_activex_button.xlsm")

```
